lib/evmone/eof.cpp

```diff
@@ -551,13 +551,25 @@ std::variant<EOF1Header, EOFValidationError> validate_eof1(
         offset += code_size;
     }
 
-    std::unordered_set<uint16_t> accessed_code_sections = {0};
+    std::vector<bool> visited_code_sections(code_sizes.size());
+    std::queue<uint16_t> code_sections_worklist;
+
+    code_sections_worklist.push(0);
```

Comment:
```suggestion
    std::queue<uint16_t> code_sections_worklist{0};
```

Reply:
I initially tried that but got this error:

```
error: no matching function for call to ‘std::queue<short unsigned int>::queue(<brace-enclose
d initializer list>)’                                                                                                                                                    
  555 |     std::queue<uint16_t> code_sections_worklist{0};
```